api/spec/v1.yaml

```diff
@@ -910,7 +990,50 @@ components:
           description: Whether this transaction successfully executed.
       description: |
         A consensus transaction.
-    
+
+    ConsensusEventList:
+      type: object
+      properties:
+        transactions:
+          type: array
+          items:
+            $ref: '#/components/schemas/ConsensusEvent'
+      description: |
+        A list of consensus events.
+
+    ConsensusEvent:
+      type: object
+      properties:
+        block:
+          type: integer
+          format: int64
+          description: The block height at which this event was generated.
+          example: *block_height_1
+        tx_index:
```

Comment:
Not introduced here, but now I can't unsee that we name `tx_index` here and `txn_index` in the DB. https://github.com/oasisprotocol/oasis-indexer/blob/d48de379c258fc467e09f33ecd614c03bfb8847d/storage/migrations/01_oasis_3_consensus.up.sql#L39

Reply:
I noticed that too :) and was unsure which one to go with. But eventually I decided it makes more sense to use `tx_` and be consistent with just about everything except those weirdly-named DB fields. I now sent out https://github.com/oasisprotocol/oasis-indexer/pull/252 to rename the DB field also.